specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2024-11-01/virtualMachineImage.json

```diff
@@ -860,7 +853,10 @@
           "200": {
             "description": "OK",
             "schema": {
```

Comment:
[ARMBlockingComment] This violates the contract for list APIs.. The previous definition was correct..

Reply:
please see https://github.com/Azure/azure-rest-api-specs/pull/33958#issuecomment-2809971663

Reply:
this is same operation as VirtualMachineImages_List but with "$expand=properties" added. 
VirtualMachineImages_List is a legacy operation and returns an array instead of pageable. 
Feature owner wants to return different return type when $expand is added. But want to keep same list format as the orignal VirtualMachineImages_List

Reply:
We check the policy definitions and dont see an impact on azure policy due to the revert of the response schema